mlrun/datastore/base.py

```diff
@@ -665,9 +665,10 @@ def http_head(url, headers=None, auth=None):
     return response.headers
 
 
-def http_put(url, data, headers=None, auth=None):
+def http_put(url, data, headers=None, auth=None, session=None):
```

Comment:
This method should be moved to `V3ioStore` (which is the only class that uses it), and it should always reuse the session to avoid recreating the HTTP connection.

Reply:
Created https://jira.iguazeng.com/browse/ML-5554